nginx/conf.d/prod.nginx.conf

```diff
@@ -37,6 +37,7 @@ map $uri $cache_control {
 
 add_header Cache-Control $cache_control;
 
+proxy_cache_path /etc/nginx/cache keys_zone=SC_CACHE:30m levels=1:2 inactive=30d max_size=1g;
```

Comment:
We might want the cache to be bigger than 1g, that sounds close to what the all the offline builds would be, 5g  would be safe, at the moment we have 33g available on the server.